API_README.md

```diff
@@ -828,30 +829,20 @@ Request:
 curl -X DELETE 'http://localhost:8300/apis/api/patient/1/medical_problem/1'
 ```
 
-#### POST /api/patient/:pid/allergy
+#### GET /api/allergy
 
 Request:
 
 ```sh
-curl -X POST 'http://localhost:8300/apis/api/patient/1/allergy' -d \
```

Comment:
In cases where api endpoints already exist, we should use same strategy as encounter enpoints where update the all the related endpoints to new api wiring and use uuid (for example, where patient is in these endpoints). Then would build the related fhir endpoints on top of that.

Reply:
Okay, these endpoints can easily with `getAll` service :)

Reply:
Updated the potential fix. Lemme know if that looks good. Will Update the Doc too :)

Reply:
looks good